test/value-link.test.ts

```diff
@@ -19,6 +19,7 @@ test("supports custom model", (t) => {
 		config: {
 			...customModelBase.config,
 			allowTargetBlank: true as const,
+			text: { type: "Text" as const },
```

Comment:
💡 #idea: Generating a key text field is probably more accurate and avoid the use of `as const`.

```suggestion
			text: model.keyText(),
```

Reply:
Done ✅